pxr/imaging/hd/rendererPlugin.cpp

```diff
@@ -60,7 +60,14 @@ HdRendererPlugin::~HdRendererPlugin() = default;
 HdPluginRenderDelegateUniqueHandle
 HdRendererPlugin::CreateDelegate(HdRenderSettingsMap const& settingsMap)
 {
-    if (!IsSupported()) {
+    TfToken backendToken;
```

Comment:
On the whole, we're not a fan of the changes to this function. For one, the changes mean that CreateDelegate can fail even though IsSupported might succeed (such as if you gave it no hgi token).

Would it be possible to put the switch at the UsdImagingGLEngine level? For  example, you can pass your Hgi of choice to UsdImagingGLEngine in an HdDriver.